CHANGELOG.md

```
@@ -746,56 +932,78 @@ Other changes:
 * Remove `custom_scope_suffix` parameter of `Logfire.log` by @alexmojaki in https://github.com/pydantic/logfire/pull/399
-* Add missing `service_version` field to `_LogfireConfigData` so that it gets copied into subprocesses by @alexmojaki in https://github.com/pydantic/logfire/pull/401
+* Add missing `service_version` field to `_LogfireConfigData` so that it gets copied into subprocesses by @alexmojaki
+  in https://github.com/pydantic/logfire/pull/401
 
 ## [v0.51.0] (2024-08-22)
 
 ### BREAKING CHANGES
 
-* **System metrics are no longer collected by default** when the correct dependency is installed. Use [`logfire.instrument_system_metrics()`](https://logfire.pydantic.dev/docs/integrations/system-metrics/) to enable system metrics collection. **If you are simply using the old 'Basic System Metrics' dashboard, then no further code changes are required, but that dashboard will no longer work properly and you should create a new dashboard from the template named 'Basic System Metrics (Logfire)'**. If you were using other collected metrics, see the documentation for how to collect those. By @alexmojaki in https://github.com/pydantic/logfire/pull/373
+* **System metrics are no longer collected by default** when the correct dependency is installed. Use [
+  `logfire.instrument_system_metrics()`](https://logfire.pydantic.dev/docs/integrations/system-metrics/) to enable
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside link text

(MD039, no-space-in-links)